hack/test-cmd.sh

```diff
@@ -722,6 +723,109 @@ __EOF__
+  kube::test::get_object_assert rc "{{range.items}}{{$id_field}}:{{end}}" ''
+  # Command
+  kubectl create -f examples/https-nginx/nginx-app.yaml "${kube_flags[@]}"
+  # Post-condition: nginxsvc service is running
+  kube::test::get_object_assert services "{{range.items}}{{$id_field}}:{{end}}" 'kubernetes:nginxsvc:'
+  # Post-condition: my-nginx rc is running
+  kube::test::get_object_assert rc "{{range.items}}{{$id_field}}:{{end}}" 'my-nginx:'
+  # Command
+  kubectl get -f examples/https-nginx/nginx-app.yaml "${kube_flags[@]}"
+  kubectl describe -f examples/https-nginx/nginx-app.yaml "${kube_flags[@]}"
+  # Command
+  kubectl replace -f hack/testdata/nginx-app-modify.yaml --force "${kube_flags[@]}"
+  # Post-condition: nginxsvc service and mock rc are replaced
+  kube::test::get_object_assert 'services nginxsvc' "{{${labels_field}.status}}" 'replaced'
+  kube::test::get_object_assert 'rc my-nginx' "{{${labels_field}.status}}" 'replaced'
+  # Command
+  kubectl label -f examples/https-nginx/nginx-app.yaml labeled=true "${kube_flags[@]}"
+  # Post-condition: nginxsvc service and my-nginx rc are labeled
+  kube::test::get_object_assert 'services nginxsvc' "{{${labels_field}.labeled}}" 'true'
+  kube::test::get_object_assert 'rc my-nginx' "{{${labels_field}.labeled}}" 'true'
+  # Command
+  kubectl annotate -f examples/https-nginx/nginx-app.yaml annotated=true "${kube_flags[@]}"
+  # Post-condition: nginxsvc service and my-nginx rc are annotated
+  kube::test::get_object_assert 'services nginxsvc' "{{${annotations_field}.annotated}}" 'true'
+  kube::test::get_object_assert 'rc my-nginx' "{{${annotations_field}.annotated}}" 'true'
+  # Cleanup service and rc
+  kubectl delete -f examples/https-nginx/nginx-app.yaml "${kube_flags[@]}"
+
+  ### Create, get, describe, replace, label, annotate, and then delete service nginxsvc and replication controller my-nginx from JSON, with a List type
+  # Pre-condition: no service (other than default kubernetes services) or replication controller is running
+  kube::test::get_object_assert services "{{range.items}}{{$id_field}}:{{end}}" 'kubernetes:'
+  kube::test::get_object_assert rc "{{range.items}}{{$id_field}}:{{end}}" ''
+  # Command
+  # TODO: remove --validate=false when PR "Add validate support for list kind #14726" is merged
+  kubectl create -f hack/testdata/multi-resource-list.json --validate=false "${kube_flags[@]}"
+  # Post-condition: mock service is running
+  kube::test::get_object_assert services "{{range.items}}{{$id_field}}:{{end}}" 'kubernetes:mock:'
+  # Post-condition: mock rc is running
+  kube::test::get_object_assert rc "{{range.items}}{{$id_field}}:{{end}}" 'mock:'
+  # Command
+  # kubectl create -f hack/testdata/multi-resource.json "${kube_flags[@]}" # test fails here now
+  # TODO: test get when geting List is fixed
+  # kubectl get -f hack/testdata/multi-resource-list.json "${kube_flags[@]}"
```

Comment:
We should definitely fix that. I thought we already fixed get with List, but perhaps it regressed.